frontend/src/components/common/carousel/styles.css

```diff
@@ -0,0 +1,90 @@
+html,
+body {
+  position: relative;
+  height: 100%;
+}
+
+.swiper-slide img {
+  display: block;
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.mySwiper2 {
+  width: 538px;
+  height: 376px;
+}
+.mySwiper {
+  height: 100px;
+  width: 538px;
+  box-sizing: border-box;
+  padding: 10px 0;
+  margin-top: 10px;
+}
+
+.mySwiper .swiper-slide {
+  width: 25%;
+  height: 100%;
+  opacity: 0.5;
+}
+
+.mySwiper .swiper-slide-thumb-active {
+  opacity: 1;
+}
+
+.mySwiper .swiper-slide-thumb-active .swiper-underline {
+  position: absolute;
+  bottom: 0;
+  height: 3px;
+  width: 98%;
+  background-color: blue;
+  border-bottom-left-radius: 30%;
+  border-bottom-right-radius: 30%;
+}
+
+.mySwiper2 {
+  border-radius: 10px;
+}
+.mySwiper .swiper-slide img {
+  border-radius: 3px;
+}
+
+.mySwiper2 .swiper-button-prev::after, .mySwiper2 .swiper-button-next::after {
+  visibility: hidden;
+}
+.mySwiper2:hover .swiper-button-prev::after, .mySwiper2:hover .swiper-button-next::after {
+  visibility: visible;
+  color: blue;
+  background: rgba(255, 255, 255, .5);
+  font-size: 20px;
+  font-weight: 900;
+  padding: 10px 15px;
+  border-radius: 5px;
+}
+.mySwiper2:hover .swiper-button-next::after {
+  margin-right: 40px;
+}
+.mySwiper2:hover .swiper-button-prev::after {
+  margin-left: 40px;
+}
+
+.navigation-bar {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  z-index: 999;
```

Comment:
bad practice use so big value for z-index. In the perfect world, we should define variables for z-index and use it

I prefer to do something like this but with smaller numbers - https://github.com/twbs/bootstrap/blob/main/scss/_variables.scss#L1043-L1052